pom.xml

```diff
@@ -898,6 +898,39 @@
               <exclude>**/interpreter-setting.json</exclude>
               <exclude>**/constants.json</exclude>
               <exclude>scripts/**</exclude>
+              <exclude>**/**/*.log</exclude> 
```

Comment:
Hi @felixcheung it is already in another section (see https://github.com/apache/zeppelin/pull/1401)